src/app/(about)/overview/_components/organising-team.tsx

```diff
@@ -43,15 +43,8 @@ const team = [
     ctaLink: "#",
     content: () => {
       return (
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
-          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-          aliquip ex ea commodo consequat.
-          <br /> <br /> Duis aute irure dolor in reprehenderit in voluptate
-          velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-          occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-          mollit anim id est laborum.
+        <p> Quality Assurance Engineer 
```

Comment:
Also add `ctaLink`  something similar to https://github.com/Python-Nairobi/pynboweb/blob/62145cb91f21ccf904f63b872eabc7a151524dd7/src/app/(about)/overview/_components/organising-team.tsx#L110